chronopost_fuel_multiplier.gemspec

```diff
@@ -1,12 +1,19 @@
 Gem::Specification.new do |s|
-  s.name        = "chronopost_fuel_multiplier"
-  s.version     = "0.1.0"
-  s.date        = "2017-02-24"
-  s.summary     = "Retrieve current month's Chronopost's fuel surcharges"
-  s.description = "A simple gem to ease/automate getting this data every month"
-  s.authors     = ["Bob Maerten", "Clément Joubert"]
-  s.files       = ["lib/chronopost_fuel_multiplier.rb", "lib/plain_http_client.rb"]
-  s.email       = "bob.maerten@gmail.com"
-  s.homepage    = "https://rubygems.org/gems/chronopost_fuel_multiplier"
-  s.license     = "MIT"
-end
\ No newline at end of file
+  s.name        = 'chronopost_fuel_multiplier'
+  s.version     = '0.1.1'
+  s.date        = '2017-03-08'
+  s.summary     = 'Retrieve current month\'s Chronopost\'s fuel surcharges'
```

Comment:
I really don't like the single quotes when they make you escape simple quotes in the string :(